specification/ml/put_trained_model_vocabulary/MlPutTrainedModelVocabularyRequest.ts

```diff
@@ -48,5 +48,12 @@ export interface Request extends RequestBase {
      * @availability serverless
      */
     merges?: string[]
+
+    /**
+     * The optional vocabulary value scores if required by the tokenizer.
+     * @availability stack since=8.9.0
+     * @availability serverless
+     */
+    scores?: string[]
```

Comment:
```suggestion
    scores?: double[]
```

You may need to add `import { double } from '@_types/Numeric'`